reset_install_requirements-no-darcs-no-git.sh

```
@@ -0,0 +1,224 @@
+    if [[ $? -ne 0 ]]; then
+        printf "\033[0;31mError: Failed to extract basilisk.tar.gz\033[0m\n"
+        exit 1
+    fi
+
+    # Clean up the tar file
+    rm basilisk.tar.gz
+
+    # Apply comphy-lab patches (macOS only)
+    apply_patches "basilisk"
+
+    cd basilisk/src
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)